pkgs/applications/audio/qjackctl/default.nix

```diff
@@ -1,4 +1,8 @@
-{ stdenv, mkDerivation, fetchurl, pkgconfig, alsaLib, libjack2, dbus, qtbase, qttools, qtx11extras }:
+{ stdenv, mkDerivation, fetchurl
+, pkgconfig, alsaLib, libjack2, dbus, qtbase, qttools, qtx11extras
+# Enable jack session support
+, jackSession ? false
```

Comment:
Why is this disabled by default? It does not pull in any other dependencies.

Reply:
> The option is disabled by default for the following reasons:
> 1.The JACK-Session API is deprecated: https://jackaudio.org/api/group__JackSessionManagerAPI.html
> 2. Disabling this feature allows to use qjackctl with PipeWire